coralproject/talk
# Fix Permalink copy on iOS

## What does this PR do?
Fix Permalink copy which did not work on iOS - fixes #1927 

## How do I test this PR?

- Open Comment stream on iOS device or Simulator
- Click 'Share' next to any comment
- Click 'Copy'
- Observe copy working correctly 🎉
